### PR TITLE
configresolver dashboard: add request rate panel & ${range} var

### PR DESCRIPTION
**Pull request by AlexNPavel**
Add request rate based on the increase of the request duration histogram metric, and add an adjustable range variable.

### CODE REVIEW

**Comment by AlexNPavel** [comment]
Example in staging grafana: https://grafana-prow-monitoring-stage.svc.ci.openshift.org/d/703f0ccf02cc4339a374b52eb10f653b/ci-operator-configresolver-dashboard?orgId=1&from=1571161139812&to=1571247539813&var-range=3h

File: cluster/ci/monitoring/mixins/grafana_dashboards/configresolver.jsonnet [hunk 1]
```diff
@@ -101,20 +142,39 @@ dashboard.new(
...
     )), gridPos={
     h: 9,
     w: 24,
     x: 0,
     y: 0,
   })
+.addPanel(
+    (graphPanel.new(
+        'Config Resolver Requests Over Time',
+        description='sum(increase(configresolver_http_request_duration_seconds_bucket[${range}]))',
```
Comment (stevekuznetsov) [inline cluster/ci/monitoring/mixins/grafana_dashboards/configresolver.jsonnet]: Why `increase` and not `rate`?

Comment (AlexNPavel) [reply cluster/ci/monitoring/mixins/grafana_dashboards/configresolver.jsonnet]: It seems like rate doesn't give the actual number of requests (unless maybe I'm using incorrectly?): https://grafana-prow-monitoring-stage.svc.ci.openshift.org/d/1FNwBpTZk/ci-operator-configresolver-dashboard-rate?orgId=1&from=1571163351929&to=1571249751929&var-range=3h

Comment (AlexNPavel) [reply cluster/ci/monitoring/mixins/grafana_dashboards/configresolver.jsonnet]: I believe rate is per second increase

**Pull request merged**